ChangeLog.md

```diff
@@ -24,6 +24,15 @@ See docs/process.md for more on how version tagging works.
   static string data in your program.   This has long been part of the native
   ELF linker and should not be observable in well-behaved programs.  This
   behavior can be disabled by passing `-Wl,-O0`.
+- `EXPORT_ES6` will now emit static URLs for main WebAssembly file as well
+  as for helper Worker used by `-pthread` that can be statically detected
+  by modern bundlers at build time. In particular, you no longer have to set
+  `Module.locateFile` hook and `Module.mainScriptUrlOrBlob` settings -
+  both bundlers and browsers should pick up the required files automatically.
+  Note: this doesn't yet cover other use-cases that emit external files,
+  such as dynamic linking, `--proxy-to-worker`, external memory etc.
+- `EXPORT_ES6` can now be used in combination with `-o [filename].html`.
+- `EXPORT_ES6` no longer requires setting custom `EXPORT_NAME` too.
```

Comment:
Please add the PRs that added the features, using `(#1234)` notation. Then people can see more details and sample code there etc.

Reply:
Makes sense. I was following existing example, line above as well as several of those below doesn't have a PR attached either :P

Reply:
Done.

Reply:
Yeah the line above is a little odd since its an llvm change and there no corresponding emscripten commit .. so its a bit out of place here.. if we had a changelog for emsdk (i.e the integration of llvm with emscripten) it would be better of there (but we don't)

Reply:
Rebased & resolved conflicts. Should I re-enable auto-merge?

Reply:
yup!

Reply:
Oh, too bad other entries lack PR numbers... I think we should always add them.